app/webpacker/packs/application.js

```diff
@@ -1,9 +1,5 @@
 import './govuk_assets_import'
 import '../styles/application.scss'
-import Rails from 'rails-ujs'
-import Turbolinks from 'turbolinks'
 import { initAll } from 'govuk-frontend'
 
-Rails.start()
-Turbolinks.start()
```

Comment:
What's with these changes?

Reply:
that was the controversial commit. it removes rails ujs and turbolinks